lib/travis/build/script/r.rb

```diff
@@ -217,11 +249,15 @@ def r_github_install(packages)
           ].join(' ')
           sh.cmd "Rscript -e '#{install_script}'"
         end
-        
+
         def r_binary_install(packages)
           return if packages.empty?
-          case config[:os]
-          when 'linux'
+          if config[:os] == 'linux'
+            if config[:sudo] == false
```

Comment:
That is a good point. Do you know the approximate date when the default changed from `sudo: required` to `sudo: false`?

`unless config[:sudo]` seems a fine compromise. All `language: r` projects currently have to have sudo enabled (either by default if they are old or with an explicit `sudo: required`). It is also not really a problem if the build runs _with_ sudo, they just won't be able to take advantage of the container based infrastructure and caching.

Thank you for the review and suggestion.